flake.nix

```diff
@@ -6,6 +6,18 @@
     flake-parts.url = "github:hercules-ci/flake-parts";
     flake-parts.inputs.nixpkgs-lib.follows = "nixpkgs";
 
+    # These flakes are only used by crane at the moment, we pin them here so
+    # that flake users can override them as needed.
+    flake-utils.url = "github:numtide/flake-utils";
+    rust-overlay.url = "github:oxalica/rust-overlay";
```

Comment:
I need to make use of this for vim to be able to find rustc. But I can make an new PR for that.

Reply:
I don't understand the issue here, are you not using `nix develop` and start vim from there?

Reply:
I am yes. But then which rust toolchain will be available in my shell and to vim? There isn't one. I was thinking I would need to set `rustToolchain = pkgs.rust-bin.stable.latest.default` then pass that in nativeBuildInputs to the shell.nix.

Reply:
It does actually not use this overlay by default but just rustc from nixpkgs.